tools/create-app.bash

```diff
@@ -5,10 +5,23 @@
 
 # Params: $1: output directory
 
-# Sign app for release
+### Assets ###
+
+# Place logo in the assets file
+full_path=$(realpath $0)
+dir_path=$(dirname $full_path)
+mv ./logo-hippie.png $dir_path/../app/assets/images/
+
+# Generating splash screen
+tns resources generate splashes <./splash-image.png> --background <rgb-color>
```

Comment:
The splash image should be searched from a given images directory, as argument. Do we always need to use a color? While we don't have it we can hardcode it so that the script can run without previous edition on `<rgb-color>`